File: Bahasa C/PRAK504-2210817210013-Muhammad-ShandyDailami.c
#include <stdio.h>
int reverse(int reversed) {
    int i = 0;
    while(reversed != 0) {
        i = i * 10;
        i = i + reversed % 10;
        reversed /= 10;
    }
    return i;
}
int main() {
    int A, B, i = 0;
    while(i < 3) {
        scanf("%d %d", &A, &B); 
        A = reverse(A); 
        B = reverse(B);
        int C = A + B; 
        printf("%d", reverse(C));
    i++;
    }
    return 0;
}

File: Python/PRAK504-2210817210013-Muhammad-ShandyDailami.py
def reverse(reversed):
    i = 0
    while reversed != 0:
        i = (10 * i) + reversed % 10
        reversed //= 10
    return i

for i in range(3):
    nilai1, nilai2 = input().split()
    nilai1 = reverse(int(nilai1))
    nilai2 = reverse(int(nilai2))
    hasil = nilai1 + nilai2
    print(f'{reverse(hasil)}\n')